### DISCUSSION TITLE
MQTT vs snapshot vs latest image generation frequency

### DISCUSSION THREAD

**Discussion by Sceptersax** [blakeblackshear/frigate#7904] [Ask A Question] [answered]
This is a question about double-take but I think frigate config has something to do with it. Basically when a person is detected in frigate events, I see that a snapshot, latest and mqtt image is generated. So i know that double-take is properly subscribed to frigate. What I'm not understanding is why mqtt images are a lot more infrequent which is problematic for me since i'm depending on an mqtt event to trigger a post request in HA. 

Exactly what determines the mqtt image frequency?

**Comment by NickM-27** [comment, marked as answer]
`best_image_timeout` determines when a new best image is saved

**Reply by Sceptersax** [reply]
Thank you!
